packages/appium/docs/en/guides/security.md

```diff
@@ -2,4 +2,61 @@
+and not sharing it with anyone else, and don't expose Appium's port to the wider internet, you
+should have nothing to worry about, and can safely enable all Appium's features.
+
+But because many Appium users might not be able to guarantee such a safe environment, the Appium
+team puts many features behind a security protection mechanism which forces system admins (the
+people that are in charge of starting the Appium server) to opt-in to these features explicitly.
+(Third-party driver and plugin authors can also [hide behaviour behind security
+flags](../ecosystem/build-drivers.md).)
+
+For security reasons, Appium client sessions can _not_ request feature enablement via capabilities.
+This is the responsibility of the server admin who launches the Appium server.
+
+## Security Server Args
+
+The [Server CLI Args](../cli/args.md) doc outlines three relevant arguments which may be passed to
+Appium when starting it from the command line:
+
+|Parameter|Description|
+|----|----|
+|`--relaxed-security`|Setting this flag turns on _all_ insecure features (unless blocked by `--deny-insecure`; see below)|
+|`--allow-insecure`|Setting this flag to a comma-separated list of feature names or a path to a file containing a feature list (each name on a separate line) will allow _only_ the features listed. For example, `--allow-insecure=adb_shell` will cause _only_ the ADB shell execution feature to be enabled. This is true _unless_ `--relaxed-security` is also used, in which case all features will still be enabled. It makes no sense to combine this flag with `--relaxed-security`.|
+|`--deny-insecure`|This flag can likewise be set to a comma-separated list of feature names, or a path to a feature file. Any features listed here will be _disabled_, regardless of whether `--relaxed-security` is set and regardless of whether the names are also listed with `--allow-insecure`.|
+
+## Insecure Features
+
+Each Appium driver is responsible for its own security, and can create its own feature names. Thus
+you should read through the documentation for a particular driver to know which feature names it
+might use. Here is an incomplete list of examples from some of Appium's official drivers:
+
+|Feature Name|Description|Supported Extension(s)|
+|------------|-----------|-------|
+|`get_server_logs`|Allows retrieving of Appium server logs via the Webdriver log interface|IOS, XCUITest, Android, UiAutomator2, Espresso|
+|`adb_shell`|Allows execution of arbitrary shell commands via ADB, using the `mobile: shell` command|Android, UiAutomator2, Espresso|
+|`record_audio`|Allow recording of host machine audio inputs|XCUITest|
+|`execute_driver_script`| Allows to send a request which has multiple Appium commands.|Execute Driver Plugin|
+
+## Examples
+
+To turn on the `get_server_logs` feature for my Appium server, I could start it like this:
+
+```bash
+appium --allow-insecure=get_server_logs
```

Comment:
Will we use `appium server` or `appium` to start the appium server as a primary command in our docs?
I don't have a strong opinion for now, but maybe it would be nice to use one method consistently in our documentation. Then... probably `appium server` to follow the format in the help?

```
(headspin) kazu$ appium -h
usage: appium [-h] [-v] {server,driver,plugin} ...

A webdriver-compatible server that facilitates automation of web, mobile, and other types of apps across various platforms.

positional arguments:
  {server,driver,plugin}
    server              Run an Appium server
    driver              Access the driver management CLI commands
    plugin              Access the plugin management CLI commands

optional arguments:
  -h, --help            show this help message and exit
  -v, --version         show program's version number and exit
```

```suggestion
appium server --allow-insecure=get_server_logs
```

Reply:
that's a good question. @boneskull what do you think? i like `appium server` for explicitness, but it is not what we've used in the past and it is longer. most people probably expect `appium` to start the server since that is what it has always done.

Reply:
Yes, just use `appium`

Reply:
yeah, i think that's my vote as well.